packages/interface/src/GraphQLLanguageService.js

```diff
@@ -76,6 +78,13 @@ export class GraphQLLanguageService {
     // Perform syntax diagnostics first, as this doesn't require
     // schema/fragment definitions, even the project configuration.
     let queryHasExtensions = false;
+    if (this._extensions && this._extensions.length > 0) {
+      await Promise.all(
+        this._extensions.map(async extension => {
+          this._graphQLConfig = await extension(this._graphQLConfig);
+        }),
+      );
+    }
```

Comment:
Why are you pathing `_graphQLConfig` in every call if it's initialized in the constructor?